src/components/Game.js

```diff
@@ -17,19 +17,19 @@ import {
   letterInWord,
   letterInArray,
   getIndiciesOfLetter
-} from './Utils';
+} from '../Utils';
 
-class App extends Component {
-  constructor ({ match }) {
+class Game extends Component {
+  constructor (props) {
     super();
-    let _hideArtWork = match.params.hideArtwork === 'hard';
+    let _hideArtWork = props.hideArtwork || props.match.params.hideArtwork === 'hard';
```

Comment:
Why not use `hideArtwork` for the variable name? Then you can omit it's value on line 29.